### PR TITLE
[HUGO]Fixed things in website section

### PR DESCRIPTION
**Pull request by Guaris**

### CODE REVIEW

File: docs/websites/cms/configure-apache-to-run-multiple-wordpress-sites-on-one-linode.md [hunk 1]
```diff
@@ -130,6 +130,7 @@ Up until this point, the steps have been fairly straightforward and similar to s
 
 3.  Put the following contents in `example1.com`:
 
+ {{<file-excerpt>}}
```
Comment (GuessWhoSamFoo) [inline docs/websites/cms/configure-apache-to-run-multiple-wordpress-sites-on-one-linode.md]: Indentation, shortcode formatting

File: docs/websites/cms/install-odoo-10-on-ubuntu-16-04.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ external_resources:
 ---
 
 *This is a Linode Community guide. [Write for us](/docs/contribute) and earn $250 per published guide.*
-<hr>
+
```
Comment (GuessWhoSamFoo) [inline docs/websites/cms/install-odoo-10-on-ubuntu-16-04.md]: `---`

File: docs/websites/cms/how-to-install-ghost-cms-on-ubuntu-16-04.md [hunk 3]
```diff
@@ -82,57 +82,57 @@ Ghost-CLI is a command line interface (CLI) tool that makes installing and updat
...
-       ? Enter your MySQL password: areallysecurepassword
-       ? Enter your Ghost database name: exampleGhost
-       ? Do you wish to set up Nginx? yes
-       ? Do you wish to set up SSL? yes
-       ? Enter your email (used for Let's Encrypt notifications) user@example.com
-       ? Do you wish to set up "ghost" mysql user? yes
-       ? Do you wish to set up Systemd? yes
-       ? Do you want to start Ghost? yes
-
+{{<file>}}
```
Comment (GuessWhoSamFoo) [inline docs/websites/cms/how-to-install-ghost-cms-on-ubuntu-16-04.md]: works buy buggy

File: docs/websites/cms/install-odoo-10-on-ubuntu-16-04.md [hunk 4]
```diff
@@ -174,16 +174,16 @@ xmlrpc_port = 8069
...
-          *  `db_port = False` - Odoo uses PostgreSQL default port `5432`, change this only if you're using custom PostgreSQL settings.
-          *  `db_user = odoo` - Name of the PostgreSQL database user. In this case we used the default name, but if you used a different name when creating your user, substitute that here.
-          *  `db_password = FALSE` - Change `FALSE` to the PostgreSQL password you created previously.
-          *  `addons_path =` - Modify this line to read: `addons_path = /opt/odoo/addons`. Add `</path/to/custom/modules>` if you're using custom modules, substituting your own path.
-          *  Include the path to log files, and add a new line: `logfile = /var/log/odoo/odoo-server.log`. You can skip this line if you plan to only use `journald` for logging.
-          *  Optionally, we could include a new line specifying the Odoo Frontend port used for connection: `xmlrpc_port = 8069`. This only makes sense if you're planning to run multiple Odoo instances (or versions) on the same server. For normal installation, you can skip this line and this instance of Odoo will connect by default to port `8069`.
-
+{{<note>}}
```
Comment (GuessWhoSamFoo) [inline docs/websites/cms/install-odoo-10-on-ubuntu-16-04.md]: No note needed

File: docs/websites/ecommerce/opencart-on-centos-6.md [hunk 5]
```diff
@@ -19,12 +19,10 @@ OpenCart is an open source storefront designed to give you flexibility and fine-
 
 In order to use OpenCart, you will need to ensure that PHP is configured properly. Make sure the settings below match the values in your `/etc/php.ini` file:
 
-{{< file >}}
-/etc/php.ini
+{{< file "/etc/php.ini"  >}}
+register\_globals = Off magic\_quotes\_gpc = Off file\_uploads = 1 session\_auto\_start = 0
```
Comment (GuessWhoSamFoo) [inline docs/websites/ecommerce/opencart-on-centos-6.md]: delete slashes

File: docs/websites/ecommerce/opencart-on-debian-6-squeeze.md [hunk 6]
```diff
@@ -20,11 +20,11 @@ OpenCart is an open source storefront designed to give you flexibility and fine-
 
 In order to use OpenCart, you will need to ensure that PHP is configured properly. Make sure the settings below match the values in your `/etc/php5/apache2/php.ini` file:
 
-{{< file >}}
-/etc/php5/apache2/php.ini
+{{< file "/etc/php5/apache2/php.ini" >}}
+register\_globals = Off magic\_quotes\_gpc = Off file\_uploads = 1 session\_auto\_start = 0
```
Comment (GuessWhoSamFoo) [inline docs/websites/ecommerce/opencart-on-debian-6-squeeze.md]: slashes

**Comment by Guaris** [comment]
@GuessWhoSamFoo Do ure thing

**Pull request merged**